app/models/middleware_server_group.rb

```diff
@@ -1,6 +1,11 @@
 class MiddlewareServerGroup < ApplicationRecord
   belongs_to :middleware_domain, :foreign_key => "domain_id"
   has_many :middleware_servers, :foreign_key => "server_group_id", :dependent => :destroy
+  has_many :middleware_deployments, :foreign_key => "server_group_id", :dependent => :destroy
   serialize :properties
   acts_as_miq_taggable
+
+  def ext_management_system
+    try(:middleware_domain).try(:ext_management_system)
+  end
```

Comment:
because if we run an operation on a server group this method is being called here https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/mixins/middleware_operations_mixin.rb#L32
(`MiddlewareServerGroupController` includes the `MiddlewareOperationsMixin`)

It basically calls those method defined here: https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/hawkular/middleware_manager.rb#L256

After this PR I've already prepared a PR that adds more operations like start servers, stop servers, resume, suspend servers in the server group.